src/components/HomePage.js

```diff
@@ -8,7 +9,7 @@ const HomePage = () => {
       <NavBar />
       <div className="max-h-full flex items-center justify-center flex-col lg:flex-row">
         <div className="scale-[80%]  2xl:scale-90 xl:pl-20 flex flex-col items-center">
-          <img src="hero-image.png" className="w-full max-w-[900px] animate-float" />
+          <Image alt='This is a picture of an island with palm trees, a pineapple, coconut, surfboard, flower, starfish, shells, river, sandals, and a laptop.' src="hero-image.png" className="w-full max-w-[900px] animate-float" />
```

Comment:
Nice work! For the alt tags you don't need to be so verbose. And for images that are purely decorative like this hero image, you can have blank alt text. 

```suggestion
          <Image alt='' src="hero-image.png" className="w-full max-w-[900px] animate-float" />
```

See some examples:

https://www.w3.org/WAI/WCAG21/Techniques/html/H37

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes